layouts/partials/sidebar-tree.html

```diff
@@ -36,7 +36,9 @@
 <ul class="td-sidebar-nav__section pr-md-3">
 {{ if .section.Parent }}{{/* hide root from menu */}}
   <li class="td-sidebar-nav__section-title">
-    <a  href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $show }} collapsed{{ end }}{{ if $active}} active{{ end }}{{ if $selected}} selected{{ end }} td-sidebar-link td-sidebar-link__section">{{ $s.LinkTitle }}</a>
+    <a  href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $show }} collapsed{{ end }}{{ if $active}} active{{ end }}{{ if $selected}} selected{{ end }} td-sidebar-link td-sidebar-link__section">
```

Comment:
yeah, font awesome FTW! I can dig it.

Are we still using `collapsed`, `active` and `selected` classes now that we have an explicit use of the `fa-*` on it's own line? Maybe clean up and remove? 

I guess not really a big deal to leave them as the cost to render the static HTML is only done once on the server, and is nice to have classes in place for future CSS.  Not too concerned, but worth a quick question.

Reply:
From my understanding, `collapsed` is used for hiding the subsections, so better to keep that in -- thanks for checking!